handbook/handbook.in

```diff
@@ -143,7 +143,7 @@ Depending on the size, creating the file system can take several minutes.</p>
 
 <h2>Overview</h2>
 
-<p><img src="images/nomadbsd-overview.png" alt="" /></p>
+<p><img src="handbook/images/nomadbsd-overview.png" alt="" /></p>
```

Comment:
The link source is not correct. Either use the relative path, `images/nomadbsd-overwiew.png`, or use an absolute path, `@ROOTDIR@handbook/images/nomadbsd-overview.png`.

Reply:
Thanks for the great feedback! Fixed :-)